Cloud Application Manager/Automating Deployments/script-box.md

```diff
@@ -44,6 +45,22 @@ Can not find the one you need? Then click on **Create a new deployment policy bo
 
 The easiest way to understand script boxes is to build one. Follow this tutorial to build a simple box that says [Hello World](../Getting Started/getting-started-with-hello-world.md).
 
+### Attach script-events to Script Box
+
+Script box allows to include scripts attached to different lifecycle events (install, configure, start, stop, dispose). These scripts can be edited using UI editor.
+
+![Create script-event](../../images/cloud-application-manager/box-script/lifecycle-event1.png)
+
+![Script editor](../../images/cloud-application-manager/box-script/lifecycle-event2.png)
+
+After editing and saving, script-event allows to set a custom timeout value. This value prevents the system to limit the execution of long-time scripts.
```

Comment:
```suggestion
After you have finished editing and have saved your script, you can specify a custom timeout value in minutes for the script execution. Once you reach this value, it will trigger the abort action of the script execution. You can specify a timeout value for any script to add to the box.
```